src/vma/event/delta_timer.cpp

```diff
@@ -73,13 +73,14 @@ timer::~timer()
 	NOT_IN_USE(iter);
 	NOT_IN_USE(to_free);
 	return;
-#endif // DEFINED_VMAPOLL		
+#else
```

Comment:
@igor-ivanov Is this fix related to page faults? If not - would you consider a separate commit?